location.md

```diff
@@ -36,13 +36,19 @@ With three state universities, many tech companies ranging from small startups t
 
 ## Travel
 
-<!-- TODO: Change me -->
-Zürich is well connected to the rest of Europe with many direct flights and trains to other major European cities every day.
+### By train
+
+Paris is very easily reachable by train. There are [seven different train stations](https://en.wikivoyage.org/wiki/Paris#By_train), all of which are connected through the local transport system. Many direct trains are available from various European cities such as Amsterdam, Brussels, Cologne, Frankfurt, London, Milan, Munich, Turin, and Zurich.
+
+### By plane
+
+Paris has two international airports, **CDG** and **ORY**. The main airport, [Paris Charles de Gaulle Airport (**CDG**)](https://en.wikivoyage.org/wiki/Paris_Charles_de_Gaulle_Airport), is located in the north of Paris. This airport is connected with central Paris through train line *RER B*.
```

Comment:
If possible, I think I'd insist on suggesting people to prefer landing in this airport instead of the other one. It's a lot easier to get into Paris from this one.